### PR TITLE
Release/6.9.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
## Added

- Public: Added Authentication module as a beta feature
- Internal: Added support for testing across multiple browsers.
- Internal: Added polyfills for `Object.entries` and `Object.fromEntries` for IE11.

### Changed

- Update to module on `tsconfig.json` from `es6` to `esnext`, to allow conditional imports of specific modules (especially useful for Auth/IDV bundle separation).

### Fixed

- UI: Fix SVG icon on Cross Device Uploads Successful, Selfie Intro screens not displaying on some iOS devices, e.g. iPhone 12

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -41,7 +42,7 @@ jobs:
             ${{ runner.os }}-node-
 
       - name: Install Dependencies
-        run: npm ci
+        run: npm install
```
Comment (kopijunkie) [inline .github/workflows/CI.yml:45]: Why was this change needed for this release?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,20 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+<!-- Update date and version value -->
+
```
Comment (kopijunkie) [inline CHANGELOG.md:10]: Should remove this comment since it's already updated at line 13 below for the release.

**Review by kopijunkie — CHANGES_REQUESTED** [review]
Just 1 minor change in `CHANGELOG` & 1 question about a change just for curiosity.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (stefaniacardenas) [reply .github/workflows/CI.yml:45]: `npm ci` was breaking with this error `abbrev not accessible from nopt`  and I didn't have time to properly look into it as I need to start the applause cycle.

Comment (kopijunkie) [reply .github/workflows/CI.yml:45]: Ah, ok. Interesting. Was just wondering about it since I remember you made tweaks to the workflow to allow the `npm ci` command to run.

**Review by kopijunkie — APPROVED** [review]
